RELEASES.md

```diff
@@ -36,6 +36,11 @@
 - Added support for reading and writing byte ranges in files remotely
   ([#1795](https://github.com/enso-org/enso/pull/1795)). This allows the IDE to
   transfer files to a remote back-end in a streaming fashion.
+- Modified the `package.yaml` format in preparation for the library ecosystem
+  ([#1797](https://github.com/enso-org/enso/pull/1797)). The `engine-version`
+  field has been deprecated in favour of an `edition` field that allows to set
+  up the engine version and dependency resolution using the upcoming Edition
+  system.
```

Comment:
Which one? Because the migration happens automatically.

Or you mean that using migrated configs in old projects may lead to an unexpected engine version?

Reply:
Added a description of how the migration affects older tools.